tests/measurements/test_mutual_info.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Unit tests for the mutual_info module"""
+import numpy as np
+
+import pennylane as qml
+
+
+class TestMutualInfo:
+    """Tests for the mutual_info function"""
+
+    def test_mutual_info(self):
+        """Tests the output of qml.mutual_info"""
+        dev = qml.device("default.mixed", wires=2)
+
+        @qml.qnode(dev)
+        def circuit():
+            return qml.mutual_info(wires0=[0], wires1=[1], log_base=2)
+
+        res = circuit()
+        expected = 0
+        assert np.allclose(res, expected)
+        assert np.allclose(
+            qml.mutual_info(wires0=[0], wires1=[1], log_base=2).process_state(
+                state=dev.state, device_wires=dev.wires
+            ),
+            expected,
+        )
```

Comment:
Maybe increase the complexity of this test by including one other case (with non-zero mutual information). Also I think what we should really be testing is to determine if a `StateMeasurement` was queued and contains all of the required methods / attributes. This is because I imagine the mathematics of computing mutual information is probably well tested in the tests for `qml.math.mutual_info`

Reply:
Increased the complexity of the tests.

Added test that checks that the class `_MutualInfo` is queued.